main.css

```diff
@@ -0,0 +1,36 @@
+.topnav {
+    background-color: #333;
+    overflow: hidden;
+  }
+  
+ 
+  .topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+  }
+  
+
+  .topnav a:hover {
+    background-color: #ddd;
+    color: black;
+  }
+  
+
+  .topnav a.active {
+    background-color: #04AA6D;
+    color: white;
+  }
+
+  .content {
+    display: flex;
+    height: 500px;
+    width: 100%;
+  }
+
+  .image {
+    height: 500px;
+  }
```

Comment:
Kindly Use basic CSS to size and position them appropriately on the page. 
It may be helpful to make their backgrounds different colors so you can see the different sections